docs/bsp.md

```diff
@@ -777,11 +892,8 @@ trait CompileReport {
 ```
 
 The server is free to send any number of `build/publishDiagnostics` and `build/logMessage`
-notifications during compilation before completing the response. The client is free to forward these
-messages to the LSP editor client.
-
-The client will get a `originId` field in `CompileReport` or `CompileResult` if the `originId`
-field in the `CompileParams` is defined.
+notifications during compilation before completing the response. Any number of subtasks may be communicated
```

Comment:
Can you clarify what "subtask" means? It's mentioned three times but not explicitly introduced.